backend/src/sms/models/sms-message-transactional.ts

```diff
@@ -44,6 +44,7 @@ export class SmsMessageTransactional extends Model<SmsMessageTransactional> {
   @Column({ type: DataType.TEXT, allowNull: false })
   body: string
 
+  // indexed for faster search
   @Column({ type: DataType.STRING, allowNull: true })
```

Comment:
Can add `unique: true` here instead of commenting actually

Reply:
by right should add `@Index` decorator, but [couldn't get it to work](https://github.com/opengovsg/postmangovsg/pull/2077#issuecomment-1621530115). same issue encountered when I was adding an index for WhatsApp's `message_id` column.